metadata.rb

```diff
@@ -1,14 +1,14 @@
-# coding: UTF-8
-
 maintainer 'Cerner Corp'
 maintainer_email 'Bryan.Baugher@Cerner.com'
-license 'All rights reserved'
+license 'Apache-2.0'
 description 'Installs/Configures tomcat'
 long_description 'This cookbook is meant to install and configure an instance of tomcat'
 issues_url 'https://github.com/cerner/cerner_tomcat/issues'
 source_url 'https://github.com/cerner/cerner_tomcat'
 name 'cerner_tomcat'
 
+chef_version '>= 12.0' if respond_to?(:chef_version)
```

Comment:
the purpose of respond_to is to handle situations where the version of chef doesn't have a `chef_version` method for metadata.rb which looks to have been [added in 12.6](https://docs.chef.io/config_rb_metadata.html). Currently this cookbook and its dependencies have no problems with Chef 13, our test kitchen runs still work fine so I left it open so we don't have to tweak and release it later